### PR TITLE
Fixed wrong UV order

### PR DESCRIPTION
**Pull request by samuel100u**
Fixed wrong UV order.

### CODE REVIEW

**Comment by samuel100u** [comment]
image

**Comment by dmitryermilov** [comment]
@samuel100u , please squash commits and give  better name to git commit.  "Update common_utils.cpp" doesn't sound good.

**Review by dmitryermilov — CHANGES_REQUESTED** [review]
@samuel100u , please squash commits and give  better name to git commit.  "Update common_utils.cpp" doesn't sound good.

**Comment by dmitryermilov** [comment]
@samuel100u , please squash these 4 commits into one..

**Comment by dmitryermilov** [comment]
@samuel100u  ?

**Comment by samuel100u** [comment]
Hi I am here. Sorry I am busy this week. I will finish it!

File: tutorials/common/common_utils.cpp [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2019 Intel Corporation
+// Copyright (c) 2020 Intel Corporation
```
Comment (romka0017) [inline tutorials/common/common_utils.cpp:1]: 2019-2020

File: tutorials/common/common_utils.cpp [hunk 2]
```diff
@@ -173,12 +173,12 @@ mfxStatus LoadRawFrame(mfxFrameSurface1* pSurface, FILE* fSource)
     if (w > 2048)
         return MFX_ERR_UNSUPPORTED;
 
-    // load U
-    sts = ReadPlaneData(w, h, buf, ptr, pitch, 0, fSource);
+    // load V
+    sts = ReadPlaneData(w, h, buf, ptr, pitch, 1, fSource);
```
Comment (romka0017) [inline tutorials/common/common_utils.cpp:177]: Why do we need to have offset 1 here but 0 for U?

**Review by romka0017 — COMMENTED** [review]
And please add info to commit message: Which color format will you fix?

File: tutorials/common/common_utils.cpp (hunk 2 above)
Comment (samuel100u) [reply tutorials/common/common_utils.cpp:177]: According to [common_utils.h#L93](https://github.com/Intel-Media-SDK/MediaSDK/blob/master/tutorials/common/common_utils.h#L93), LoadRawFrame read YV12 format into NV12.
YV12 = YYYYYYYYVVUU  ,  NV12 = YYYYYYYYUVUV
That means that read YV12 with U first to NV12 is wrong in YV12 !

Comment (romka0017) [reply tutorials/common/common_utils.cpp:177]: Ok. If we read from plane format to NV12.

File: tutorials/common/common_utils.cpp (hunk 1 above)
Comment (samuel100u) [reply tutorials/common/common_utils.cpp:1]: change to 2019-2020

ok

**Review by romka0017 — COMMENTED** [review]
Please, update commit message.
Add info about color formats, which you will fix.

**Review by romka0017 — APPROVED** [review]

**Comment by samuel100u** [comment]
@dmitryermilov  Done

**Review by dmitryermilov — APPROVED** [review]

**Pull request merged**